services/digilocker-certificate-api/Dockerfile

```diff
@@ -1,7 +1,7 @@
 ############################
 # STEP 1 build executable binary
 ############################
-FROM golang:alpine AS builder
+FROM golang:1.20-alpine3.18 AS builder
```

Comment:
why did you change this?

Reply:
build was failing due to latest version on golang:alpine